adal/src/main/java/com/microsoft/aad/adal/BrokerAccountServiceHandler.java

```diff
@@ -359,7 +359,15 @@ private void bindToBrokerAccountService(final Context context, final Callback<Br
         }
         final CallbackExecutor<BrokerAccountServiceConnection> callbackExecutor = new CallbackExecutor<>(callback);
         mPendingConnections.put(connection, callbackExecutor);
-        final boolean serviceBound = context.bindService(brokerAccountServiceToBind, connection, Context.BIND_AUTO_CREATE);
+
+        boolean serviceBound = false;
+        try {
+            // Handle exception when Context#bindService fails.
+            serviceBound = context.bindService(brokerAccountServiceToBind, connection, Context.BIND_AUTO_CREATE);
+        } catch (SecurityException exception) {
```

Comment:
might be nice to be able to tie this exception into the AuthenticationException that we surface to the caller.